config.h

```diff
@@ -45,6 +47,10 @@
 #define DFU_CAN_UPLOAD      _ENABLE
 /** Handle DFU_DETACH request in DFU mode. System reset will be issued. */
 #define DFU_DETACH          _ENABLE
+/** Whether application image is verified by CRC32 */
+#define DFU_VERIFY_CHECKSUM _DISABLE
```

Comment:
I think only this one variable can be used for the checksum feature. For example _DISABLE, CRC32FAST, CRC32SMALL, ANYOTHERCHECKSUM